data/gegno/gegno ships.txt

```diff
@@ -0,0 +1,1007 @@
+	description `	This project was meant to result in two sister-ship behemoths, but the blueprints for the other design went missing early in the process, and are currently nowhere to be found.`
+
+ship Slate
+	sprite "ship/gegno slate"
+	thumbnail "thumbnail/gegno slate"
+	attributes
+		category Transport
+		licenses
+			"Gegno Civilian"
+			"Vi Evocati"
+		cost 825
+		"required crew" 1
+		bunks 7
+		mass 121
+		drag 2
+		"heat dissipation" 0.8
+		"fuel capacity" 400
+		"cargo space" 60
+		"outfit space" 122
+		"engine capacity" 58
+		weapon
+			"blast radius" 18
+			"shield damage" 40
+			"hull damage" 132
+			"hit force" 210
+	outfits
+		"C3 Scrap Cell"
+		Hyperdrive
+		"R01 Skirmish Battery"
+		"RG15 Torch Steering"
+		"RG15 Torch Thruster"
+	engine 0 40
+	leak leak 60 50
+	explode "small explosion" 5
+	explode "tiny explosion" 10
+	description `Designed by the Gegno Vi, the Slate is a modified Shale with a stronger hull and more space, a prime characteristic of Vi vessels. Slightly larger than other shuttle ships to suit the Vi's stature, it is able to carry several Gegno cadets along with a small cargo load.`
+
+ship Schist
+	sprite "ship/gegno schist a"
+	thumbnail "thumbnail/gegno schist a"
+	attributes
+		category "Light Freighter"
+		licenses
+			"Gegno Civilian"
+			"Vi Evocati"
+		cost 1932000
+		hull 12375
+		"required crew" 36
+		bunks 86
+		mass 560
+		drag 5.9
+		"heat dissipation" 0.7
+		"fuel capacity" 500
+		"cargo space" 140
+		"outfit space" 250
+		"weapon capacity" 58
+		"engine capacity" 106
+		"weapon capacity" 58
+		weapon
+			"blast radius" 71
+			"shield damage" 310
+			"hull damage" 660
+			"hit force" 700
+	outfits
+		"C7 Brawl Cell"
+		Hyperdrive
+		"Irate Cannon" 2
+		"Irate Cannon Turret"
+		"R02 Battlezone Battery"
+		"RG18 Torch Steering"
+		"RG18 Torch Thruster"
+	engine 0 153.5
+	gun -16.5 -181.5 "Irate Cannon"
+	gun 16.5 -181.5 "Irate Cannon"
+	turret 0 -139.5 "Irate Cannon Turret"
+	leak leak 60 50
+	leak flame 60 80
+	explode "large explosion" 10
+	explode "medium explosion" 25
+	explode "small explosion" 25
+	explode "tiny explosion" 10
+	"final explode" "final explosion medium"
+	description `Based on the Gegno's old Dolomite, the Schist is a mid-range ship used by the Gegno Vi. It can carry a decent amount of cargo, and has enough bunks to carry a whole platoon of Gegno warriors. It is the primary vessel used to transport recruits to the front lines, and veterans back to home territories.` 
+	description `	Although is it lightly armed compared to other Vi ships, Schists are still mounted a pair of cannons and a turret so it can bring some type of weaponry to a possible engagement.`
```

Comment:
```suggestion
	description `	Although is it lightly armed compared to other Vi ships, Schists are still mounted with a pair of cannons and a turret so it can bring some type of weaponry to a possible engagement.`
```